e2e_samples/parking_sensors/scripts/deploy_adf_artifacts.sh

```diff
@@ -47,17 +47,17 @@ adfFactoryBaseUrl="$baseUrl/resourceGroups/${RESOURCE_GROUP_NAME}/providers/Micr
 log "Deploying Data Factory artifacts."
 
 # Deploy all Linked Services
-create_adf_linked_service "Ls_KeyVault_01"
-create_adf_linked_service "Ls_AdlsGen2_01"
-create_adf_linked_service "Ls_AzureSQLDW_01"
-create_adf_linked_service "Ls_AzureDatabricks_01"
-create_adf_linked_service "Ls_Http_Parking_Bay_01"
+createLinkedService "Ls_KeyVault_01"
```

Comment:
This is not the last version - when deploying the following error arises: Deploying Data Factory artifacts.
./scripts/deploy_adf_artifacts.sh: line 50: createLinkedService: command not found. The latest version is create_adf_linked_service and should be aligned with the content of the common.sh file